packages/Webkul/Shop/src/Http/Controllers/ProductsCategoriesProxyController.php

```diff
@@ -49,12 +52,20 @@ public function index(Request $request)
 
                 $customer = auth()->guard('customer')->user();
 
-                return view($this->_config['product_view'], compact('product', 'customer'));
+                $orderCount = $this->orderItem->getModel()
```

Comment:
Move this query to the model or repository and also use customer order relation to count data.